MobileCenterUpdates/MobileCenterUpdates/MSUpdates.h

```diff
@@ -3,7 +3,7 @@
 @interface MSUpdates : MSServiceAbstract
```

Comment:
For now we only have Updates service that requires custom URL scheme. I'm pretty sure that it should be per service because core doesn't know about each service. If we have only one scheme for the SDK, core should understand what the request means and then send it over to a proper service. I'm pretty sure during this process, it will go through all started services to find an appropriate service. The benefit of simple integration would be less than expect I think.